app/serializers/dependents_serializer.rb

```diff
@@ -1,6 +1,62 @@
 # frozen_string_literal: true
 
+module DependentsHelper
+  def still_pending(award_event_id)
```

Comment:
More of a preference but I'm not sure that this needs to be a lambda here and could just be a method where you pass in the decision and award_event_id
  
```
  def still_pending?(decision, award_event_id)
    decision[:award_event_id] == award_event_id &&
      Time.zone.parse(decision[:award_effective_date]) > Time.zone.now
  end
```

and when it's called on line 40    
`decs.any? { |d| still_pending?(d, dec[:award_event_id]) }`

Reply:
I've gone ahead with this change, i had definitely overthought the lambda implementation